crates/teloxide/src/utils/render.rs

```diff
@@ -0,0 +1,249 @@
+//! Utils for rendering HTML and Markdown output.
+
+use teloxide_core::types::{MessageEntity, MessageEntityKind as MEK};
+
+use tag::*;
+
+pub use helper::RenderMessageTextHelper;
+
+mod helper;
+mod html;
+mod markdown;
+mod tag;
+
+/// Parses text and message entities to produce the final formatted output.
+#[derive(Clone, Eq, PartialEq)]
+pub struct Render<'a> {
```

Comment:
I think it's better to name it `Renderer` in the same fashion that we name builders `Builder` and not `Build`.